src/pug/blog/future_developments.pug

```diff
@@ -0,0 +1,159 @@
+extends _blog.pug
+
+block append subtitle
+  | Future developments update
+
+block content
+  h2 Future developments update
+  +news_header('6 November 2025', 'report_30', '')
+
+  h3 State of funding
+  p Small recap: When we launched Input Labs in #[em 2022] the plan was to fund the project via Patreon, later in #[em 2023] we started 3D-printing some parts and selling them in our shop, then since late 2023 we have been selling complete DIY kits in the shop, and that became our main funding source.
+  p This current funding model pays the bills, but unfortunately it #[em does not provide much income] to fund new research and development.
+  p We expected this situation to be temporary while we grew a bit more, but #[em organic growth] (without marketing investment) has been slowing down, and current #[em shipping restrictions] to the United States only made it worse.
+  p While we love #[em 3D-printing], it became a #[em bottleneck] to scale up our operations, not because 3d-printing is slow (which can be solved by having more printers) but because inconsistencies in the resulting parts require a lot of time doing additional #[em quality checks and post-processing], the kind of task that cannot be outsourced to a 3rd-party company without deep understanding of the controller.
+  p For all these reasons, we want to introduce #[em injection molding] as an additional manufacturing method for our devices, and sell #[em pre-assembled controllers] to a larger audience. But we are not abandoning 3D-printing (more on that below).
+  p During the last months we worked on putting together a #[em financial plan] to develop an injection molded Alpakka, including the molds, certification, supply chain, and potential partners (more details below).
+
+
+  hr
+  h3 Results of the Alpakka survey
+  ul
+    li Almost #[em 200 people] answered the survey about the Alpakka present and future.
+    li Average satisfaction of Alpakka owners is #[em 8.6 / 10].
+
+  p: b Most loved features in the Alpakka 1:
+  ul
+    li Gyro performance.
+    li Open source and open hardware.
+    li Scrollwheel.
+
+  p: b Less loved features in the Alpakka 1:
+  ul
+    li Build quality.
+    li Firmware install process.
+    li Missing advanced features in the Ctrl app.
+
+
+  p: b Most hyped features for the Alpakka 2:
+  ul
+    li Buy pre-assembled.
+    li Quad-gyro.
+    li Analog triggers.
+
+
+  hr
+  h3 DIY and PRO versions
+  p Moving forward, Input Labs' devices will have 2 variants:
+
+  table.manual.vdiv.col2
+    tr
+      th DIY version
+      th PRO version
+    tr
+      td: div
+        span Our classic approach to Do-It-Yourself, for makers that own a 3D-printer.
+        p PCB and electronic components available in the shop.
+        p 3d-printed parts will NOT be available in the shop anymore.
```

Comment:
You may want to clarify that they will be replaced by corresponding injection-molded parts.